definitions/aircraft/Asobo_TBM930.yaml

```diff
@@ -36,6 +36,12 @@ shared:
     type: var
     var_name: L:XMLVAR_Pitot_2
     var_type: f64
+    
+  - # ASOBO_DEICE_Switch_Pitot_Template
+    type: ToggleSwitch
+    var_name: A:PITOT HEAT
+    var_units: Bool
+    event_name: PITOT_HEAT_TOGGLE
```

Comment:
Not needed. The XMLVAR of Pitot 1/2 already calls PITOT HEAT upon toggle.

Reply:
From testing, this def is required otherwise the switches do not move.

Reply:
deice.yaml is already defined so this is a duplicate.

Reply:
true that